src/Symfony/Component/Cache/Traits/PdoTrait.php

```diff
@@ -140,6 +140,33 @@ public function createTable()
     /**
      * {@inheritdoc}
      */
+    public function prune()
+    {
+        $pdoConn = $this->getConnection();
+        $selects = $pdoConn->prepare("SELECT $this->idCol FROM $this->table WHERE $this->lifetimeCol + $this->timeCol <= :time");
```

Comment:
Of course, you are right; I'd pulled the from another method, but that one had a need to also return values. :-) Only three lines now.